scripts/tests/chiptest/test_definition.py

```diff
@@ -64,17 +64,11 @@ def stop(self):
             return True
         return False
 
-    def reboot(self, discriminator):
```

Comment:
@vivien-apple I didn't see this get used anywhere

Reply:
Right. This is probably a leftover. In the first version that I wrote there were no "manager" class like "Accessories" which is now the one calling (start/stop).